core-tests/src/test/java/org/mule/runtime/core/api/util/ClassUtilsTestCase.java

```diff
@@ -236,14 +268,78 @@ public void getInheritedFieldValueWithoutRecurse() throws Exception {
     ClassUtils.getFieldValue(new ExtendedHashBlob(1), "hash", false);
   }
 
+  @Test
+  public void getStaticFieldValue() throws Exception {
+    final List<String> hashBlobProperties = Arrays.asList(new String[] {"one", "two"});
+    HashBlob.setStaticHashProperties(hashBlobProperties);
+    List<String> value = ClassUtils.getStaticFieldValue(HashBlob.class, "staticHashProperties", false);
+    assertEquals(hashBlobProperties, value);
+  }
+
+  @Test
+  public void getStaticFieldValueRecursive() throws Exception {
+    final List<String> hashBlobProperties = Arrays.asList(new String[] {"one", "two"});
+    ExtendedHashBlob.setStaticHashProperties(hashBlobProperties);
```

Comment:
static import

Reply:
Cannot import static method due collision in method name with HashBlob class,